### ISSUE TITLE
Test failures with icalendar 5.0.9

### ISSUE THREAD

**Issue by dotlambda** [niccokunzmann/x-wr-timezone#8]
<details>

```
============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-7.4.0, pluggy-1.2.0
rootdir: /build/source
collected 278 items

test/test_command_line.py .                                              [  0%]
test/test_convert_examples.py FFF......FFFFFF...FFFFFFssssssssssssssssss [ 15%]
ssssssssssssssssssssssssssssssFFF.........FFFFFF...FFFssssssssssssssssss [ 41%]
ssssssssssssssssssssssssssssss................................ssssssssss [ 67%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssss...ssssss...       [ 91%]
test/test_copy.py ......................                                 [ 98%]
test/test_examples.py ..                                                 [ 99%]
test/test_readme.py .                                                    [100%]

=================================== FAILURES ===================================
_______________ test_input_to_output[to_standard-calendar_pair0] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <rdate hackerpublicradio.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Hacker Public Radio
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
_______________ test_input_to_output[to_standard-calendar_pair1] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <rdate hackerpublicradio.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Hacker Public Radio
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
_______________ test_input_to_output[to_standard-calendar_pair2] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <rdate hackerpublicradio.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Hacker Public Radio
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
_______________ test_input_to_output[to_standard-calendar_pair9] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <moved event RECURRENCE ID.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Partyborn Zeitgeist
E       assert VCALENDAR({'P...NSPARENT'')})) == VCALENDAR({'P...NSPARENT'')}))
E         Omitting 7 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair10] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <moved event RECURRENCE ID.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Partyborn Zeitgeist
E       assert VCALENDAR({'P...NSPARENT'')})) == VCALENDAR({'P...NSPARENT'')}))
E         Omitting 7 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair11] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <moved event RECURRENCE ID.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Partyborn Zeitgeist
E       assert VCALENDAR({'P...NSPARENT'')})) == VCALENDAR({'P...NSPARENT'')}))
E         Omitting 7 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair12] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <x wr timezone not present.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: test
E       assert VCALENDAR({'V...fff66447c0>})) == VCALENDAR({'V...fff6645720>}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair13] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <x wr timezone not present.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: test
E       assert VCALENDAR({'V...fff65c6440>})) == VCALENDAR({'V...fff65c4df0>}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair14] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <x wr timezone not present.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: test
E       assert VCALENDAR({'V...fff6644a30>})) == VCALENDAR({'V...fff6a05330>}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair18] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <single event no tz.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: MyCalendar
E       assert VCALENDAR({'V...timezone'')})) == VCALENDAR({'V...timezone'')}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair19] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <single event no tz.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: MyCalendar
E       assert VCALENDAR({'V...timezone'')})) == VCALENDAR({'V...timezone'')}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair20] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <single event no tz.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: MyCalendar
E       assert VCALENDAR({'V...timezone'')})) == VCALENDAR({'V...timezone'')}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair21] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <exdate hackerpublicradio modified.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Hacker Public Radio
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair22] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <exdate hackerpublicradio modified.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Hacker Public Radio
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
______________ test_input_to_output[to_standard-calendar_pair23] _______________

to_standard = <function to_standard at 0x7ffff6a78280>
calendar_pair = <exdate hackerpublicradio modified.in+.out>

    def test_input_to_output(to_standard, calendar_pair):
        """Test the calendars which are a pair of input and output."""
        output = to_standard(calendar_pair.input.as_icalendar())
>       assert output == calendar_pair.output.as_icalendar(), calendar_pair.message
E       AssertionError: Hacker Public Radio
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:17: AssertionError
___________ test_output_stays_the_same[to_standard-output_calendar0] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <exdate hackerpublicradio modified.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
___________ test_output_stays_the_same[to_standard-output_calendar1] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <exdate hackerpublicradio modified.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
___________ test_output_stays_the_same[to_standard-output_calendar2] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <exdate hackerpublicradio modified.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar12] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <rdate hackerpublicradio.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar13] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <rdate hackerpublicradio.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar14] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <rdate hackerpublicradio.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...ity News'')})) == VCALENDAR({'V...ity News'')}))
E         Omitting 4 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar15] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <moved event RECURRENCE ID.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'P...NSPARENT'')})) == VCALENDAR({'P...NSPARENT'')}))
E         Omitting 7 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar16] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <moved event RECURRENCE ID.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'P...NSPARENT'')})) == VCALENDAR({'P...NSPARENT'')}))
E         Omitting 7 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar17] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <moved event RECURRENCE ID.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'P...NSPARENT'')})) == VCALENDAR({'P...NSPARENT'')}))
E         Omitting 7 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar21] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <x wr timezone not present.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...fff649dea0>})) == VCALENDAR({'V...fff649d3f0>}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar22] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <x wr timezone not present.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...fff65d4400>})) == VCALENDAR({'V...fff65d47c0>}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
__________ test_output_stays_the_same[to_standard-output_calendar23] ___________

to_standard = <function to_standard at 0x7ffff6a78280>
output_calendar = <x wr timezone not present.out>

    def test_output_stays_the_same(to_standard, output_calendar):
        assert output_calendar.is_corrected_output()
        output = to_standard(output_calendar.as_icalendar())
>       assert output == output_calendar.as_icalendar(), "A calendar that was modified one should stay as it is."
E       AssertionError: A calendar that was modified one should stay as it is.
E       assert VCALENDAR({'V...fff65dbaf0>})) == VCALENDAR({'V...fff65d87c0>}))
E         Omitting 5 identical items, use -vv to show
E         Use -v to get more diff

test/test_convert_examples.py:23: AssertionError
=========================== short test summary info ============================
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair0] - AssertionError: Hacker Public Radio
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair1] - AssertionError: Hacker Public Radio
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair2] - AssertionError: Hacker Public Radio
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair9] - AssertionError: Partyborn Zeitgeist
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair10] - AssertionError: Partyborn Zeitgeist
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair11] - AssertionError: Partyborn Zeitgeist
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair12] - AssertionError: test
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair13] - AssertionError: test
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair14] - AssertionError: test
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair18] - AssertionError: MyCalendar
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair19] - AssertionError: MyCalendar
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair20] - AssertionError: MyCalendar
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair21] - AssertionError: Hacker Public Radio
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair22] - AssertionError: Hacker Public Radio
FAILED test/test_convert_examples.py::test_input_to_output[to_standard-calendar_pair23] - AssertionError: Hacker Public Radio
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar0] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar1] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar2] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar12] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar13] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar14] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar15] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar16] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar17] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar21] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar22] - AssertionError: A calendar that was modified one should stay as it is.
FAILED test/test_convert_examples.py::test_output_stays_the_same[to_standard-output_calendar23] - AssertionError: A calendar that was modified one should stay as it is.
================== 27 failed, 85 passed, 166 skipped in 2.07s ==================
```
</details>

<small>
<hr/>
We're using <a href="https://polar.sh/niccokunzmann">Polar.sh</a> so you can upvote and help fund this issue. We receive the funding once the issue is completed & confirmed by you. Thank you in advance for helping prioritize & fund our work.
</small>

<a href="https://polar.sh/niccokunzmann/x-wr-timezone/issues/8">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/niccokunzmann/x-wr-timezone/issues/8/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/niccokunzmann/x-wr-timezone/issues/8/pledge.svg">
</picture>
</a>

**Comment by niccokunzmann** [comment]
Thanks for reporting this! I wonder what caused the breakage. I think that maybe the tests need adjusting a little but actually, the functionality is still intact. python-recurring-ical-events would fail otherwise. It is good to look into it and port the CI to Github Actions to catch these kinds of errors.
What are your thoughts?

**Comment by niccokunzmann** [comment]
Thanks for working on it. That is better. I think that a PR to fix this might have a version of icalendar above a certain version in the requirements. I wonder which element makes it fail.

**Comment by niccokunzmann** [comment]
I am opening it again until the tests pass. #9 makes an attempt.

**Comment by niccokunzmann** [comment]
@dotlambda, #9 makes the test intact again. The icalendar update helped compare the calendars and find a bug. I will close this for now. Feel free to open it again if you have any doubts!

**Comment by niccokunzmann** [comment]
The fix is live as v0.0.6.
